moodle/tests/moodlestandard_test.php

```diff
@@ -436,4 +436,69 @@ public function test_squid_php_commentedoutcode() {
         // Let's do all the hard work!
         $this->verify_cs_results();
     }
+
+    public function test_moodle_files_moodleinternal_problem() {
+        $this->set_standard('moodle');
+        $this->set_sniff('moodle.Files.MoodleInternal');
+        $this->set_fixture(__DIR__ . '/fixtures/moodle_files_moodleinternal/problem.php');
+
+        // Define expected results (errors and warnings). Format, array of:
+        // - line => number of problems,  or
+        // - line => array of contents for message / source problem matching.
+        // - line => string of contents for message / source problem matching (only 1).
+        $this->set_errors(array());
+        $this->set_warnings(array());
```

Comment:
little detail, this shouldn't be here, we are setting warning immediately below.